detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsLambdaDetectorSync.ts

```diff
@@ -19,47 +19,32 @@ import {
   IResource,
   Resource,
   ResourceAttributes,
-  ResourceDetectionConfig,
 } from '@opentelemetry/resources';
-import {
-  SEMRESATTRS_CLOUD_PROVIDER,
-  SEMRESATTRS_CLOUD_PLATFORM,
-  SEMRESATTRS_CLOUD_REGION,
-  SEMRESATTRS_FAAS_VERSION,
-  SEMRESATTRS_FAAS_NAME,
-  CLOUDPROVIDERVALUES_AWS,
-  CLOUDPLATFORMVALUES_AWS_LAMBDA,
-} from '@opentelemetry/semantic-conventions';
 
 /**
  * The AwsLambdaDetector can be used to detect if a process is running in AWS Lambda
  * and return a {@link Resource} populated with data about the environment.
  * Returns an empty Resource if detection fails.
  */
 export class AwsLambdaDetectorSync implements DetectorSync {
-  detect(_config?: ResourceDetectionConfig): IResource {
+  detect(): IResource {
+    const awsRegion = process.env.AWS_REGION;
     const functionName = process.env.AWS_LAMBDA_FUNCTION_NAME;
-    if (!functionName) {
-      return Resource.empty();
-    }
-
     const functionVersion = process.env.AWS_LAMBDA_FUNCTION_VERSION;
-    const region = process.env.AWS_REGION;
+    const logGroupName = process.env.AWS_LAMBDA_LOG_GROUP_NAME;
+    const logStreamName = process.env.AWS_LAMBDA_LOG_STREAM_NAME;
+    const memorySize = process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE as string;
 
     const attributes: ResourceAttributes = {
```

Comment:
Any reason you are preferring to hardcode the attribute keys and not use the key values from `@opentelemetry/semantic-conventions`?

Reply:
Actually, some of them never existed in `@opentelemetry/semantic-conventions` because the module couldn't keep up with the docs update.